azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -732,6 +733,19 @@ public void shutdownNow() {
     logger.warn("Shutting down FlowRunnerManager now...");
     this.executorService.shutdownNow();
     this.triggerManager.shutdown();
+    deleteExecutionDirectory();
```

Comment:
I don't see the shutdownNow method calling the shutdown method anywhere. But maybe it'd be better if we didn't delete the execution directory on shutdownNow because logs that were only partially created would be lost. What do you think?